chart/install.yaml

```diff
@@ -44,6 +44,7 @@ data:
       metricsAddr: "127.0.0.1:8080"
       leaderElection: true
       metricsSink: "noop"
+      slackTokenFilePath: "/etc/chaos-slack-token"
```

Comment:
This should be autogenerated from the chart but I can't see anything specifying this in the chart

Reply:
Also I would probably not expect any notifiers to be enabled by default or eventually the noop one.